### PR TITLE
docs:  Removed team references that do not exist.

### PR DESCRIPTION
**Pull request by sharepointoscar**
*Description of changes:*

1. The `teams` folder content now matches the instructions on the **readme**
2. The `envs` folder referenced files on the **readme** now match actual file names.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: envs/dev/templates/team-carmen.yaml [hunk 1]
```diff
@@ -1,21 +0,0 @@
-apiVersion: argoproj.io/v1alpha1
```
Comment (kcoleman731) [inline envs/dev/templates/team-carmen.yaml]: why are we dropping these?

Comment (sharepointoscar) [reply envs/dev/templates/team-carmen.yaml]: @kcoleman731  - The file is not part of the teams referenced on the readme of this repo nor the main SSP workshop repo.  I want to make sure that the workshop teams matches the teams also on this repo...

Comment (kcoleman731) [reply envs/dev/templates/team-carmen.yaml]: Team carmen is in the readme? https://github.com/aws-samples/ssp-eks-workloads/blob/main/README.md?plain=1#L50

Team carmen also provides and example of an Application pointing to a remote source url (i.e. not in this repo) so it is a good example to keep. 

I think it is ok if the workshop only references two of the four teams that are in this repo.

File: README.md [hunk 2]
```diff
@@ -61,20 +59,21 @@ The configuration in the `teams` subdirectories represent the individual teams t
...
 Each `team` subdirectory in turn has environment subdirectories. The configuration in those directories are Kubernetes manifests that represent the individual workload that is running for each team and in each environment. 
 
+**NOTE** Some team environment directories may contain additional `yaml` files such as an `ingress.yaml`
 ```
 teams
 ├── team-burnham
 │   ├── dev
-│   │   ├── guestbook-ui-deployment.yaml
-│   │   └── guestbook-ui-svc.yaml
+│   │   ├── deployment.yaml
```
Comment (kcoleman731) [inline README.md:68]: this we need for sure.

File: envs/dev/templates/team-carmen.yaml (hunk 1 above)
Comment (sharepointoscar) [reply envs/dev/templates/team-carmen.yaml]: Ok added the `carmen` manifests under `dev` environments, and to the readme on this repo.

**Review by kcoleman731 — APPROVED** [review]

**Pull request merged**